Dockerfile

```diff
@@ -1,4 +1,15 @@
-FROM node:5.1.1
+FROM ubuntu
```

Comment:
We should really be using the `node:4` as that is what is current LTS version of node.

For stability of the project it best to follow the nodejs LTS releases.

https://github.com/nodejs/LTS#lts_schedule